### ISSUE TITLE
Failed to parse source map from '(...)/src/tailwind.tsx`

### ISSUE THREAD

**Issue by D1no** [MathiasGilson/Tailwind-Styled-Component#42]
Hi, first of all: Amazing Project! Thank you for building this!

So, when I run `npm start` the console spits out: can't find src/ folder but still serves the react app correctly o.0

```bash
WARNING in ./node_modules/tailwind-styled-components/dist/tailwind.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/worker/development/hundertschaft-com/node_modules/tailwind-styled-components/src/tailwind.tsx' file: Error: ENOENT: no such file or directory, open '/Users/worker/development/hundertschaft-com/node_modules/tailwind-styled-components/src/tailwind.tsx'
 @ ./node_modules/tailwind-styled-components/dist/index.js 14:35-56
 @ ./src/App.tsx 7:0-44 9:19-24
 @ ./src/index.tsx 7:0-24 11:33-36

3 warnings have detailed information that is not shown.
Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.

webpack 5.68.0 compiled with 3 warnings in 3036 ms
No issues found.
```

Not sure if simply publishing the `src` folder to npm would fix this.

## Reproduction

- As Github Repo: https://github.com/D1no/reproduction-tailwind-styled-component
- As Codesandbox: https://codesandbox.io/s/pensive-carson-y45qv?file=/src/App.tsx
   _(resize the separator between browser and terminal in codesandbox to get back the full build log)_

Steps
- Follow [tailwindcss-create react app steps](https://tailwindcss.com/docs/guides/create-react-app)
- `npm i -D tailwind-styled-components`

Thanks!

**Comment by rickyzhangca** [comment]
sam error, but using `"import tw from 'tailwind-styled-components";` works for me and `import tw from 'tailwind-styled-components';` does not

**Comment by rickyzhangca** [comment]
can confirm, happening to me as well on a fresh create-react-app project

**Comment by rickyzhangca** [comment]
my current workaround is setting up `react-app-rewired` and in `config-overrides.js`, put 

```
// webpack configs

const { override } = require('customize-cra');

const ignoreWarnings = value => config => {
    config.ignoreWarnings = value;
    return config;
};

module.exports = override(
    ignoreWarnings([/Failed to parse source map/]),
  );
```

to ignore the warning so it's more friendly to pipeline stuffs

**Comment by MathiasGilson** [comment]
do you have this issue with the package version 2.1.2 ?

**Comment by rickyzhangca** [comment]
> do you have this issue with the package version 2.1.2 ?

tried @D1no 's sandbox with 2.1.2, same errors

![image](https://user-images.githubusercontent.com/16908811/155008754-fc0708d8-f72a-419d-a275-1553ca7c7181.png)

**Comment by MathiasGilson** [comment]
Should be fixed in the last package version 2.1.6

**Comment by arminhupka** [comment]
Unfortunately error still exist.
